index.js

```diff
@@ -58,6 +59,14 @@ class WebdriverAjax {
     function disableInterceptor() {
       return browser.executeAsync(interceptor.disableInterceptor);
     }
+
+    function excludeUrls(urls) {
+      urls = urls.map(function(regex) {
+        return typeof regex === 'object' ? regex.source : regex
+      })
+      return browser.executeAsync(interceptor.excludeUrls, urls)
```

Comment:
This converts everything input into a string, ignoring RegExp flags, etc (like case (in)sensitivity.